packages/date-picker/src/DatePicker/DatePickerMenu/DatePickerMenu.spec.tsx

```diff
@@ -135,15 +135,23 @@ describe('packages/date-picker/date-picker-menu', () => {
     });
     test('select menu triggers have aria labels', () => {
       const result = renderDatePickerMenu();
-      const monthSelect = result.getByLabelText('Select month');
-      const yearSelect = result.getByLabelText('Select year');
+      const monthSelect = result.getByLabelText('Select month', {
```

Comment:
super-nit (only if you have other things to fix): 
extract `{ getByLabelText } = renderDatePickerMenu` so these lines prettify more nicely